src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4555,7 +4567,9 @@ void ApplicationManagerImpl::AddAppToRegisteredAppList(
   LOG4CXX_DEBUG(
       logger_,
       "App with app_id: " << application->app_id()
-                          << " has been added to registered applications list");
+                          << " has been added to registered applications list. "
+                             "List size after added: "
```

Comment:
@ydementieiev `List size after added: ` -> `New list size: `

Reply:
@AByzhynar [thank you ](https://github.com/LuxoftSDL/sdl_core/pull/44/commits/eb9f13417feaa48a59ba6824dfc4becbaf7e3dea#diff-6289bfe1988a528c2f109887d563c017R4571)